docs/releases.md

```diff
@@ -0,0 +1,76 @@
+# Ceph CSI Operator driver Release Process
+
+- [Ceph CSI Operator driver Release Process](#ceph-csi-operator-driver-release-process)
+  - [Introduction](#introduction)
+  - [Versioning](#versioning)
+  - [Tagging repositories](#tagging-repositories)
+  - [Release process \[TBD\]](#release-process-tbd)
+
+## Introduction
+
+This document provides details about Ceph CSI operator release process.
+
+## Versioning
+
+The ceph-csi-operator project uses
+[semantic versioning](http://semver.org/)
+for all releases.
+Semantic versions are comprised of three
+fields in the form:
+
+```MAJOR.MINOR.PATCH```
+
+For examples: `1.0.0`, `1.0.0-rc.2`.
+
+Semantic versioning is used since the version
+number is able to convey clear information about
+how a new version relates to the previous version.
+For example, semantic versioning can also provide
+assurances to allow users to know when they must
+upgrade compared with when they might want to upgrade:
```

Comment:
Why do we use hard newlines in an `md` file? We should leave word wrapping to the client we present the md file

Reply:
Reformatted this one but having rules helps in review also

Reply:
Even after you reformatted, the situation is not really better 
![image](https://github.com/ceph/ceph-csi-operator/assets/14993359/f5104ab8-bd42-4bd8-a3ac-5c478d23e14a)

The point is that whatever column count we choose, under a specific view it breaks. 
The only way, I am aware of, this can be solved is to never break mid-sentences and allow sentences that need to be longer than the X column, to just be.